pinc/page_controls.inc

```diff
@@ -0,0 +1,152 @@
+<?php
+include_once($relPath.'stages.inc'); // $Round_for_round_id_
+include_once($relPath.'metarefresh.inc');
+
+function get_page_data()
+{
+    $projectid = array_get($_GET, "project", "");
```

Comment:
These should all be arguments to `get_page_data()`. Ideally only the top-level `.php` pages should be accessing the `$_GET` and `$_POST` variables.